proxy/api/src/http/project.rs

```diff
@@ -111,6 +112,18 @@ fn discover_filter(
         .and_then(handler::discover)
 }
 
+/// `POST /track/<peer_id>`
```

Comment:
```suggestion
/// `PUT /<id>/track/<peer_id>`
```

Somehow the endpoints in here became inconsistent. All endpoints regarding a single resource should be bundled under that resource, the id/urn should be a param as seen in the get filter. And the peer id as well, as the docs currently lie, indicating that the peer id is part of the path.

I'd also suggest this to be a put to indicate that it's idempotent.

Reply:
Nice :+1: Good suggestions :)